tokio-util/Cargo.toml

```diff
@@ -44,8 +44,8 @@ pin-project-lite = "0.1.4"
 slab = { version = "0.4.1", optional = true } # Backs `DelayQueue`
 
 [dev-dependencies]
-tokio = { version = "0.3.0", path = "../tokio", features = ["full"] }
-tokio-test = { version = "0.3.0", path = "../tokio-test" }
+tokio = { version = "0.3.0", features = ["full"] }
+tokio-test = { version = "0.3.0" }
```

Comment:
We keep the `dev-dependencies` as path deps.

Reply:
Fixed.